XMPP Newletter March 2023 (#1262)
xsf/xmpp.org: content/blog/2023-04-05-newsletter.md

date: 2023-04-05
categories: ['Newsletter']
---

Welcome to the XMPP Newsletter, great to have you here again! This issue covers the month of March 2023.
Many thanks to all our readers and all contributors!

Like this newsletter, many projects and their efforts in the XMPP community are a result of people’s voluntary work. If you are happy with the services and software you may be using, please consider saying thanks or help these projects! Interested in supporting the Newsletter team? Read more [at the bottom](#help-us-to-build-the-newsletter).

## XSF Announcements

- xmpp.org got a new software section! Looking for XMPP software, i.e. clients, servers, libraries, components, and tools? Check out [xmpp.org](https://xmpp.org/)'s new [software section](https://xmpp.org/software/), which lets you filter software by your own criteria. Looking for a client which works on Android and supports audio/video calls? Looking for a library that supports [XEP-0461: Message Replies](https://xmpp.org/extensions/xep-0461.html)? Just apply the filter and see what you get!

{{< figure src="/images/newsletter/2023-03-05/xmpp-software-section.png" caption="xmpp.org's new software section" >}}

> - If you are interested to become an [XSF member](https://xmpp.org/community/membership/) you can [apply in this quarter until May 21th, 2023, 00:00 UTC](https://wiki.xmpp.org/web/Membership_Applications_Q2_2023).

Reviewer: ```suggestion
- If you are interested to become an [XSF member](https://xmpp.org/community/membership/) you can [apply during this quarter until May 21th, 2023, 00:00 UTC](https://wiki.xmpp.org/web/Membership_Applications_Q2_2023).
```

- [Bifrost bridge: Offline messages exchange between Matrix and XMPP](https://opencollective.com/bifrost-mam)
- [Mellium Co-op](https://opencollective.com/mellium)

## XMPP Events

- [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours): available on our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)
- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month

## Articles

- Debian package maintainers give [a summary](https://xmpp-team.pages.debian.net/blog/2023/03/xmpp-whats-new-in-bookworm.html) of XMPP packages that will be available in the next Debian stable release (Debian 12 "Bookworm").
- Cheogram Android [introduces Stickers](http://blog.jmp.chat/b/cheogram-android-stickers-2023).
- IgniteRealtime: [Translations everywhere!](https://discourse.igniterealtime.org/t/translations-everywhere/92628)

> - [Here’s Why You Should Build Scalable and Concurrent Applications with Elixir](https://www.erlang-solutions.com/blog/heres-why-you-should-build-scalable-and-concurrent-applications-with-elixir/) by Cara May-Cole

Reviewer: This has nothing to do with XMPP directly?

- Carl Schwan writes on the [Digital Market Act workshop in Brussels](https://carlschwan.eu/2023/03/02/digital-market-act-workshop-in-brussels/)
{{< figure src="/images/newsletter/2023-04-05/dma_brussels.jpg" caption="Digital Market Act workshop in Brussels" >}}

- The IETF has approved [publication of Messaging Layer Security (MLS)](https://www.ietf.org/blog/mls-secure-and-usable-end-to-end-encryption/), a new standard for end-to-end security that will make it easy for apps to provide the highest level of security to their users. End-to-end encryption is an increasingly important security feature in Internet applications. It keeps users’ information safe even if the cloud service they’re using has been breached.

## Software news
### Clients and applications

> - [Cheogram Android 2.12.1-4](https://git.singpolyma.net/cheogram-android/refs/2.12.1-4) released with support for sending message replies and reactions, swipe to reply, "note to self" contact, viewing vcard4 profiles, and occasional MUC ping.

Reviewer: ```suggestion
- [Cheogram Android 2.12.1-4](https://git.singpolyma.net/cheogram-android/refs/2.12.1-4) has been released with support for sending message replies and reactions, swipe to reply, "note to self" contact, viewing vcard4 profiles, and occasional MUC pings.
```

- [SiskinIM 7.3.3](https://github.com/tigase/siskin-im/releases/tag/7.3.3) and [BeagleIM 5.3.4](https://github.com/tigase/beagle-im/releases/tag/5.3.4) from [Tigase](https://tigase.org/) have been released.
- [Spark 3.0.2](https://igniterealtime.org/projects/spark/) from [Ignite Realtime](https://igniterealtime.org/) has been released.

{{< figure src="/images/newsletter/2023-04-05/movim_replies.png" caption="Movim supports replies" >}}

### Servers

- Metronome IM [4.0.2](https://github.com/maranda/metronome/releases/tag/v4.0.2) brings some bug fixes.

### Libraries & Tools

- [python-nbxmpp 4.2.1 and 4.2.2](https://dev.gajim.org/gajim/python-nbxmpp/-/blob/master/ChangeLog) have been released, which fix some bugs with python-nbxmpp's HTTP library.
- [QXmpp 1.5.1](https://github.com/qxmpp-project/qxmpp/releases/tag/v1.5.1) and [QXmpp 1.5.2](https://github.com/qxmpp-project/qxmpp/releases/tag/v1.5.2) have been released in February and [QXmpp 1.5.3](https://github.com/qxmpp-project/qxmpp/releases/tag/v1.5.3) in March! They fix some important bugs.
- [xmppbl.org](https://xmppbl.org) provides a Real-Time Block List (RTBL) service for XMPP.

> - [XmppDotNet](https://www.ag-software.net/2023/03/22/xmppdotnet-announcement/) is the new name and next generation of MatriX vNext XMPP library.

Reviewer: ```suggestion
- [XmppDotNet](https://www.ag-software.net/2023/03/22/xmppdotnet-announcement/) is the new name and next generation of the MatriX vNext XMPP library.
```
